plugins/SitesManager/Model.php

```diff
@@ -240,7 +240,7 @@ public function getSiteFromId($idSite)
      * Returns the list of all the website IDs registered.
      * Caller must check access.
      *
-     * @return array The list of website IDs
+     * @return int[] The list of website IDs
```

Comment:
That actually might not be correct. Depending on the mysql configuration it might be an array of strings. But guess we could add a cast to int in the foreach to ensure only ints are returned

Reply:
I could do `int[]|string[]`. As long as it's not `array`, it'll be more useful.

Reply:
Sure. no preference. Just wanted to mention that the return type can vary. Maybe casting to int would be even better, so we can rely on the return type?

Reply:
Might be more useful, but given we don't do that for any other query in Matomo (that I know of), I'm not sure it's really needed (and might make things more inconsistent).

Reply:
Feel free to change the doc block only then. Having more strict return types is a more global thing for sure 🙈